client/css/style.css

```diff
@@ -1614,8 +1624,8 @@ kbd {
 	animation-fill-mode: forwards;
 	-webkit-animation-timing-function: ease-in;
 	animation-timing-function: ease-in;
-	-webkit-animation-delay: .4s;
-	animation-delay: .4s;
+	-webkit-animation-delay: .1s;
+	animation-delay: .1s;
```

Comment:
No, we cannot 😅

We would really appreciate if you fix this so we can 👍 and merge this. Thanks! :)